### ISSUE TITLE
Unable to disable line or next line check in JSX

### ISSUE THREAD

**Issue by brenna** [sider/goodcheck#130]
I'm unable to get disable line comments working in JSX.  The only form of comments allowed are [`{/* */}`](https://reactjs.org/docs/faq-build.html#how-can-i-write-comments-in-jsx), so I've tried both:

```js
<p>problematic line</p> {/* goodcheck-disable-line */}
```

and 

```js
{/* goodcheck-disable-next-line */}
<p>problematic line</p> 
```

and even

```js
{/* // goodcheck-disable-next-line */}
<p>problematic line</p> 
```

for good measure but my goodcheck linter (run via `goodcheck check`) is still flagging the problem lines in all cases.

Is there an alternate syntax that will work for JSX or can the checker be adjusted to accommodate this?

**Comment by yykamei** [comment]
Thank you for reporting an issue! Perhaps, it might be a bug. `DISABLE_LINE_PATTERNS` and `DISABLE_NEXT_LINE_PATTERNS` in `Buffer` are intended to detect the pattern `/* ... */`, but the pattern is written without escaping asterisk `*`. I think the following patch resolves the issue. I try to create a PR :muscle: 

```diff
diff --git a/lib/goodcheck/buffer.rb b/lib/goodcheck/buffer.rb
index 2999d05..292c37d 100644
--- a/lib/goodcheck/buffer.rb
+++ b/lib/goodcheck/buffer.rb
@@ -8,7 +8,7 @@ module Goodcheck
       /# goodcheck-disable-line$/, # Ruby, Python, PHP, ...
       /-- goodcheck-disable-line$/, # Haskel, SQL, ...
       /<!-- goodcheck-disable-line -->$/, # HTML, Markdown, ...
-      /\/* goodcheck-disable-line *\/$/, # CSS, SCSS,
+      /\/* goodcheck-disable-line *\/$/, # CSS, SCSS,
       /<%# goodcheck-disable-line %>$/, # ERB, ...
       /' goodcheck-disable-line$/, # VB
     ].freeze
@@ -18,7 +18,7 @@ module Goodcheck
       /# goodcheck-disable-next-line$/, # Ruby, Python, PHP, ...
       /-- goodcheck-disable-next-line$/, # Haskel, SQL, ...
       /<!-- goodcheck-disable-next-line -->$/, # HTML, Markdown, ...
-      /\/* goodcheck-disable-next-line *\/$/, # CSS, SCSS,
+      /\/* goodcheck-disable-next-line *\/$/, # CSS, SCSS,
       /<%# goodcheck-disable-next-line %>$/, # ERB, ...
       /' goodcheck-disable-next-line$/, # VB
     ].freeze
```

**Comment by yykamei** [comment]
@brenna I released the new version `v2.5.2`, including this fix. Thank you for your report :pray: If your problem is not solved, please reopen the issue.

**Comment by brenna** [comment]
Thanks so much for the quick fix @yykamei ! I will give it a try.
